service/controller/internal/adapter/guest_instance.go

```diff
@@ -94,6 +93,8 @@ func (i *GuestInstanceAdapter) Adapt(config Config) error {
 
 		i.Master.EtcdVolume.Name = key.EtcdVolumeName(config.CustomObject)
 
+		i.Master.Ignition = base64.StdEncoding.EncodeToString([]byte(rendered))
```

Comment:
This is going to be the "small" cloudconfig hash - is this actually enough for us to detect a change? I am not convinced here. I believe that this will not register a change if we e.g. change the k8s version - only if we change the s3URL.

Reply:
Sorry for the misleading changes here. I am only renaming `CloudConfig` to `Ignition`. This is otherwise unchanged. The hashing is done here https://github.com/giantswarm/aws-operator/blob/c9aa006d112c98ac61a5f56352ff6c2ae9da619e/service/controller/resource/s3object/desired.go#L99

Reply:
I just wanted to align with the naming we've been using like with `ignition-operator`.